pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1531,6 +1578,25 @@ func (cwft *TxnComputationWrapper) SetDatabaseName(db string) error {
 
 func (cwft *TxnComputationWrapper) GetColumns() ([]interface{}, error) {
 	var err error
+	switch cwft.GetAst().(type) {
+	case *tree.ShowCreateTable:
+		cols := []*plan2.ColDef{
+			{Name: "Table"},
+			{Name: "Create Table"},
+		}
+		columns := make([]interface{}, len(cols))
+		for i, col := range cols {
```

Comment:
the code block here is duplicate with the one below starts from 1601. It is unnessary